srcs/pressure_controller.cpp

```diff
@@ -657,17 +727,36 @@ PressureController::pidPatient(int32_t targetPressure, int32_t currentPressure,
     // Compute error
     int32_t error = targetPressure + PID_PATIENT_SAFETY_PEEP_OFFSET - currentPressure;
 
-    // Compute integral
-    patientIntegral = patientIntegral + ((PID_PATIENT_KI * error * dt) / 1000000);
-    patientIntegral = max(PID_PATIENT_INTEGRAL_MIN, min(PID_PATIENT_INTEGRAL_MAX, patientIntegral));
+    if (error > -10) {
+        // change of state
+        if (patientPIDFastMode) {
+            patientIntegral = -1000;
+        }
+        patientPIDFastMode = false;
+    }
+
+    // Dont compute derivative
+    int32_t derivative = 0;
```

Comment:
Why keeping `PID_PATIENT_KD` if we don't compute derivative at all? Why not continue computing derivative and have `PID_PATIENT_KD = 0` in faulhaber?